Terminal.Gui/Core/View.cs

```diff
@@ -1899,25 +1900,27 @@ public virtual void LayoutSubviews ()
 			get => textFormatter.Text;
 			set {
 				textFormatter.Text = value;
-				if (textFormatter.Size != Bounds.Size && (((width == null || width is Dim.DimAbsolute) && Bounds.Width == 0)
-					|| ((height == null || height is Dim.DimAbsolute) && Bounds.Height == 0))) {
-					Bounds = new Rect (Bounds.X, Bounds.Y, textFormatter.Size.Width, textFormatter.Size.Height);
-					if (width == null) {
-						width = Bounds.Width;
-					} else if (width is Dim.DimAbsolute) {
-						width = Math.Max (Bounds.Width, height.Anchor (Bounds.Width));
-					}
-					if (height == null) {
-						height = Bounds.Height;
-					} else if (height is Dim.DimAbsolute) {
-						height = Math.Max (Bounds.Height, height.Anchor (Bounds.Height));
-					}
-				}
+				ResizeView (autoSize);
 				SetNeedsLayout ();
 				SetNeedsDisplay ();
 			}
 		}
 
+		/// <summary>
+		/// Used by <see cref="Text"/> to resize the view's <see cref="Bounds"/> with the <see cref="TextFormatter.Size"/>.
```

Comment:
Can you add documentation explaining that this should only be used with Absolute layout; it will not work with Computed layout?

Reply:
Ok. Done!